README.md

```diff
@@ -135,11 +135,13 @@ The `main` branch is built and tested regularly, considered as production-ready;
 
 ## Minimum Supported Rust Version policy (MSRV)
 
-The `Mercury` crate's minimum supported rust version is 1.55.0.
+The `Mercury` crate's minimum supported rust version is 1.55.1.
 
 ---
 
 ## Documentations
 
 - [Mercury API Documentation](core/rpc/README.md)
+- [Mercury Architecture Documentation](docs/architecture.md)
 - [Mercury Config Documentation](docs/config.md)
+- [Mercury Layout Documentation](docs/layout.md)
```

Comment:
Should we append contribution doc here?

Reply:
In my opinion, the documents listed here are related to mercury, and a should be placed in a separate section, reference the [usage in ckb](https://github.com/nervosnetwork/ckb#how-to-contribute) .